swupd/manifest_test.go

```diff
@@ -342,7 +342,7 @@ func TestWriteManifestFile(t *testing.T) {
 	if err != nil {
 		t.Fatal("unable to open file for write")
 	}
-	defer os.Remove(f.Name())
+	//	defer os.Remove(f.Name())
```

Comment:
remove?

Reply:
The problem as I see it is that with the 'defer' you get a message saying that the test has failed because there are differences, but when you come to look the file has been removed, so there is no evidence left, which makes it a frustrating experience.